src/screens/shared/farmSearch/FarmSearch.js

```diff
@@ -12,24 +12,50 @@ class FarmSearch extends React.PureComponent {
   constructor(props) {
     super(props);
     this.state = {
-      farms: []
+      farms: [],
+      search: null,
+      filteredFarms: []
     };
   }
 
   async componentDidMount() {
     const farms = await getAllFarmsForFarmSearch();
-    this.setState({ farms });
+    this.setState({ farms, filteredFarms: farms });
   }
 
-  render() {
+  searchAndFilter = e => {
     const { farms } = this.state;
+    const keyword = e.target.value;
+    const filteredFarms = farms.filter(
+      farm =>
+        (farm.contactFirstName &&
+          farm.contactFirstName
+            .toLowerCase()
+            .includes(keyword.toLowerCase())) ||
```

Comment:
this case may not be needed, since you're already checking if `${farm.contactFirstName} ${farm.contactLastName}` contains the keyword

Reply:
ahh yes makes sense!